ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -167,6 +167,8 @@ private OptimizerInstance getAuthenticatedOptimizer(String authToken) {
 
   @Override
   public OptimizingTask pollTask(String authToken, int threadId) {
+    OptimizerInstance optimizer = getAuthenticatedOptimizer(authToken);
+    LOG.debug("Optimizer executor[{},{}] poll task", optimizer.getToken(), threadId);
```

Comment:
This log line is a bit redundant. It has already been printed in the outer layer.